.github/workflows/daily-qa.yml

```diff
@@ -0,0 +1,65 @@
+# This workflow runs our daily scheduled acceptance tests. It runs each test suite against the
+# development head (e.g. main) and all officially supported branches.
+#
+# As this is meant to be run via scheduling, the workflow itself only runs on the default branch
+# (e.g. main), so any changes you make will affect any other branches we test via this workflow.
+name: Daily tests
+
+on:
+  workflow_dispatch: { }
+  schedule:
+    # Runs at 01:00 every day; see this link for more: https://crontab.guru/#0_1_*_*_*
+    - cron: '0 1 * * *'
```

Comment:
:thinking: Could this interfere with others?

Reply:
I've removed the scheduling of the normal `qa-testbench`, which will be explicitly triggered, so I dont' think so. Maybe the one from Testbench? I'll disable it now.